cmd/utils/flags.go

```diff
@@ -159,6 +160,12 @@ var (
 		Name:  "lightkdf",
 		Usage: "Reduce key-derivation RAM & CPU usage at some expense of KDF strength",
 	}
+	// Hard coded chain contracts
+	ReleaseOracleFlag = cli.StringFlag{
```

Comment:
Why is this a flag? I don't think private networks are concerned with versioning and if they explicitly need to set something they might as well just completely ignore it.

Unless you have some other argument for having this is a flag I think this should be removed.